PokemonGoF/PokemonGo-Bot
# Handle Github Download Zip Format

To test:

1. pull this branch
2. Put this in your config.json, replacing the tasks you currently have:
```
    "plugins": [
      "TheSavior/test-pgo-plugin#2d54eddde33061be9b329efae0cfb9bd58842655"
    ],
    "tasks": [
      {
        "type": "test-pgo-plugin.PrintText"
      }
    ],
```
3. Check that your console looks like this:
```
2016-08-08 10:33:55,360 [PokemonGoBot] [INFO] 
2016-08-08 10:34:01,156 [PokemonGoBot] [INFO] [bot_start] Starting bot...
PrintText!
PrintText!
PrintText!
PrintText!
PrintText!
PrintText!
PrintText!
PrintText!
...
```
